### ISSUE TITLE
Research how to extend the deep learning approach

### ISSUE THREAD

**Issue by sydneyvert** [dj311/uob-summer-project#21]

**Comment by sydneyvert** [comment]
[FdNN]
https://www.nature.com/articles/s41598-018-34833-6

**Comment by sydneyvert** [comment]
This one may be harder to implement:
https://www.cv-foundation.org/openaccess/content_iccv_2015/papers/Kontschieder_Deep_Neural_Decision_ICCV_2015_paper.pdf

**Comment by sydneyvert** [comment]
More like combining neurals to make a forest:
https://arxiv.org/pdf/1604.07143v1.pdf
>Methods start on page 11

**Comment by sydneyvert** [comment]
https://www.scirp.org/journal/PaperInformation.aspx?PaperID=81453
> "We first processed the features as we did in Random Forest and Neural Network algorithms. Then we ran the Random Forest classifier using the most suitable parameters as we tried several combinations of them before. In our training, we calculated the predicted class, a vote by trees in the forest weighted by their probability estimates, among which we picked the one with the highest mean probability estimate across the trees as one of our features. Training the updated features through Neural Networks, we found a slight improvement in accuracy. We tried both sequences and it turned out that there were slight improvements after such an operation."

**Comment by sydneyvert** [comment]
https://towardsdatascience.com/building-a-convolutional-neural-network-cnn-in-keras-329fbbadc5f5
